clang/include/clang/Basic/Builtins.td

```diff
@@ -1484,6 +1484,18 @@ def ElementwiseSubSat : Builtin {
   let Prototype = "void(...)";
 }
 
+def ElementwiseClz : Builtin {
+  let Spellings = ["__builtin_elementwise_clz"];
+  let Attributes = [NoThrow, Const, CustomTypeChecking, Constexpr];
+  let Prototype = "void(...)";
+}
+
+def ElementwiseCtz : Builtin {
+  let Spellings = ["__builtin_elementwise_ctz"];
+  let Attributes = [NoThrow, Const, CustomTypeChecking, Constexpr];
```

Comment:
There is no constexpr implementation in this patch.

Reply:
Oh good catch, thank you. I must have copy/pasted the wrong builtins..

Reply:
Ah no, the documentation says that the integer builtins may be called in a constexpr context, so `Constexpr` is correct and the patch needs to account for that. Will update.

Reply:
Done - it now supports constexpr evaluation.